less/index.less

```diff
@@ -57,3 +57,185 @@ html, body {
     height: 100%;
 }
 
+/*variables*/
+@font-color: dimgrey;
+@med-size-font: 2rem;
+
+/*mixins*/
+.font-varela(){
+	font-family: 'Varela Round', sans-serif;
+} 
+.flex-mixin(){
+	display: flex;
+	justify-content: space-between;
+}
+
+/*this helps me figure out the boudaries*/
+*{
```

Comment:
when a project is complete and wildcard selector is no longer needed for the borders this should be deleted from your code along with any commented out code that is not needed for a cleaner look.